fastlane/Helpers/Build.swift

```diff
@@ -0,0 +1,82 @@
+//
+//  Build.swift
+//  FastlaneRunner
+//
+//  Created by Su T. on 23/09/2022.
+//  Copyright © 2022 Nimble. All rights reserved.
+//
+
+final class Build {
+
+    static func adHoc(env: Environment) {
+        build(env: env, type: .adHoc)
+    }
+
+    static func appStore() {
+        build(env: .production, type: .appStore)
+    }
+
+    static private func build(
+        env: Environment,
+        type: BuildType
+    ) {
+        buildApp(
+            scheme: .userDefined(env.scheme),
+            clean: .userDefined(true),
+            outputName: .userDefined(env.productName),
+            includeSymbols: .userDefined(true),
+            includeBitcode: .userDefined(type == .appStore),
+            exportMethod: .userDefined(type.value),
+            exportOptions: .userDefined([
+                // NOTE: bundleId should be `env.bundleId` instead of `Constant.bundleId`
+                // To test ios-template, uncomment right below
```

Comment:
If we test ios-template with `production` env, we don't need to uncomment this, right?

Reply:
Yes, you're right. Since our staging bundle id for Staging also is the same as production, so only Staging needs that hard coded.